scripts/globals/spells/bluemagic/feather_tickle.lua

```diff
@@ -37,7 +37,7 @@ spell_object.onSpellCast = function(caster, target, spell)
         spell:setMsg(xi.msg.basic.MAGIC_TP_REDUCE)
     end
 
-    return tp
+    return 0
```

Comment:
I wonder if here should be power instead of 0, so it returns a number to show on chat?

Reply:
If a value is returned, I'm pretty sure its handled as damage.  I didn't want to return the possibility of feather tickle dealing up to 3k damage as a BLU

Reply:
fair enough